src/Tests/Runtime/config-tests/ConfigurationSerializationTests.AuxOptions.json

```diff
@@ -1,4 +1,5 @@
 {
+  "dotvvmVersion": "4.0.0.0",
```

Comment:
We should replace it in the tests, otherwise it will make the tests fail every time we change the version.

Reply:
Yes, I replaced the value with `"dotvvmVersion": "*.*.*.*"`